src/consensus/consensus.h

```diff
@@ -22,4 +34,94 @@ enum {
     LOCKTIME_MEDIAN_TIME_PAST = (1 << 1),
 };
 
+/**
+ * Context-independent CTransaction validity checks.
+ * Nobody should spend an extra cycle on a transaction that doesn't pass this.
```

Comment:
I fail to see how this additional comment is move only. You may as well just keep it the way it was, as this comment is not adding much information.

Also two lines below this you are changing the white space. `CValidationState &state`.

Still, Concept ACK